types/mithril/test/test-route.ts

```diff
@@ -125,4 +125,10 @@ route.set('/test2', undefined, {
 
 const path: string = route.get();
 
-const fn = route.link(h('div', 'test'));
+const fn1: (e?: Event) => any = route.link(h('div', 'test'));
+
+const fn2: (v: Vnode<any, any>) => any = route.link({replace: true});
+
+const ex1 = h('a', {href: '/url', oncreate: route.link});
+
+const ex2 = h('a', {href: '/url', oncreate: route.link({replace: true})});
```

Comment:
Also, it'd be worth adding tests for all the other route options, similar to `m.route.set`.